site/layouts/_default/examples.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="{{ .Site.Params.authors }}">
-    <meta name="generator" content="Hugo {{ .Hugo.Version }}">
+    <meta name="generator" content="Hugo {{ .Site.Hugo.Version }}">
```

Comment:
Not sure if this 100% right even though it works. Maybe `{{ hugo.Version }}` is the proper way @bep?

Reply:
Yes.